MvcMusicStore/Web.config

```diff
@@ -30,6 +31,21 @@
   -->
   <system.web>
     <compilation debug="true" targetFramework="4.8"/>
+    <sessionState
+      mode="SQLServer"
+      sqlConnectionString="ASPState"
+      cookieless="false"
+      timeout="20" />
+    <!--
+    Since we will be deploying in a load-balanced configuration, we must
+    ensure we have the same machineKey element on all servers
+    -->
+    <machineKey decryptionKey="1B4CC725253C394C4ED6BE3D4AF597929B5CAB45B8F46FC7" validationKey="1A77C7E084ED0E69D1BED6407A290DE667A2BAEEBA52A49E3828DD6356D1D1D29F480C01916E9769812095C00CACC58AEF3DA4091E7725E01A26D4DAFD9ABBB6" />
```

Comment:
Is it safe to commit? If not, we could [set it](https://docs.microsoft.com/en-us/dotnet/api/system.web.configuration.machinekeysection.decryptionkey?view=netframework-4.8) from an environment variable.